react/PaymentMethod.tsx

```diff
@@ -82,7 +83,10 @@ const PaymentMethod: FunctionComponent<Props & InjectedIntlProps> = ({
         </div>
         {isBankInvoice && payment.url && (
           <div className="mt5">
-            <ButtonLink to={payment.url} variation="primary" openNewWindow>
+            <ButtonLink
+              to={parseBankInvoiceUrl(payment.url)}
+              variation="primary"
+              openNewWindow>
```

Comment:
```suggestion
              openNewWindow
             >
```

Reply:
Will format the project in a future PR, it's waaaay outdated.